src/go/rpk/pkg/config/weak_test.go

```diff
@@ -834,6 +834,7 @@ func TestSeedServer(t *testing.T) {
 }
 
 func TestConfig_UnmarshalYAML(t *testing.T) {
+	intPtr := func(i int) *int { return &i }
```

Comment:
Should there be a test case without a `node_id` here?

Reply:
Done